ruby/guessing-game/guessing_game.rb

```diff
@@ -0,0 +1,47 @@
+require 'io/console'
+
+class Game
+  attr_reader :guesses, :word, :line, :guessed_arr, :line_arr
+  def initialize(word)
+    @word = word.split("")
+    @guesses = word.length * 2
+    @line_arr = @word.map {|x| x = "_"}
+    @line = @line_arr.join(" ")
+    @guessed_arr = []
+    @replace_arr = []
```

Comment:
You could consider seeding your replace array on initialize.  This could help dry up some code.

`@replace_arr = Array.new(word.length, "_")`